TelegramBot.py

```diff
@@ -27,10 +27,11 @@ def __init__(self, name, port, counter_limit, token, chat_id, recheck_duration_p
         self.token = token
         self.chat_id = chat_id
         self.recheck_duration_para = recheck_duration_para
+        self.owner = owner + " "
```

Comment:
This is a bit hacky. A cleaner solution would be to use Python's string interpolation function below where you assemble the message string:
```py
message = "{} The following server isn't responding properly. Please check it: {}".format(self.owner, self.name)
```
Obviously, you'd then need to use string interpolation instead of string concatenation also in the debug messages, too.